### ISSUE TITLE
Midpoint Review - Accessibility Feedback - IIR, Paperless Delivery

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/va.gov-team#115008]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Should:** To ensure the correct heading hierarchy, "Communications settings" on the profile page should be an h2, not an h3. The mockup might be mislabeled, check the live Profile page to confirm.
- [x] **Should:** For the paperless checkbox: if there will be more checkboxes in the future, will an alert show up every time a setting changes? As more checkboxes are added, consider using a save button to avoid too many alerts.
- [x] **Should:** The research plan should include users of assistive technology, like screen reader users.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [x] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app

**Comment by raquelou** [comment]
The mockup was mislabeled with an h3 and should have been an h2. I updated the design.

**Comment by raquelou** [comment]
In a future iteration we will consider adding a save button to reduce the number of alerts displayed. For this first iteration we will keep the similar functionality to what exists in the notification settings.

**Comment by raquelou** [comment]
Because our prototype was built in figma, which is difficult to test with users who use screen readers and zoom, we did not conduct our research with users of assistive technology. We do plan to have the designs reviewed by ADE to ensure that our design will work for users of assistive tech.
